wireapp/wire-ios
# chore: remove phone registration flow SQCORE-1295

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQCORE-1295" title="SQCORE-1295" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />SQCORE-1295</a>  [iOS] Remove phone registration flow
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Registering by phone number has been blocked by the backend for some time. This means the phone registration flow is broken and can never succeed.

### Causes

When trying to register with a phone number, the backend will return an error.

### Solutions

Remove the phone registration flow. Note, we still allow login with phone number for users who have already registered.


### Testing

#### Test Coverage (Optional)

- Updated snapshot tests.

#### How to Test

In the app, try to create a new account. See that only email registration is possible.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
